pkg/cli/admin/inspect/pod.go

```diff
@@ -75,7 +75,8 @@ func (o *InspectOptions) gatherContainerInfo(destDir string, pod *corev1.Pod, co
 	}
 
 	if err := o.gatherContainerEndpoints(path.Join(destDir, "/"+container.Name), pod, &container, port); err != nil {
-		return err
+		log.Printf("        Skipping container endpoint collection for pod %q container %q: %v\n", pod.Name, container.Name, err)
```

Comment:
`skipping` might sound misleading given it could have already collected some of the endpoints

Reply:
"skipping one or more" ;-)